include/devicetree.h

```diff
@@ -1462,6 +1575,9 @@
 #define DT_INST_PHA_BY_NAME(inst, pha, name, cell) \
 	DT_PHA_BY_NAME(DT_DRV_INST(inst), pha, name, cell)
 
+#define DT_INST_PHA_BY_NAME_OR(inst, pha, name, cell, default_value) \
+	DT_PHA_BY_NAME_OR(DT_DRV_INST(inst), pha, name, cell, default_value)
+
```

Comment:
No documentation for this one? Was that intentional?

Reply:
> No documentation for this one? Was that intentional?

It was not. Fixed and rebased, thanks.